.ci/setup.sh

```diff
@@ -71,9 +71,40 @@ else  # Linux
         sudo apt-get install --no-install-recommends -y \
             libboost1.74-dev \
             ocl-icd-opencl-dev
+    fi
+    if [[ $TASK == "gpu" || $TASK == "bdist" ]]; then
         if [[ $IN_UBUNTU_LATEST_CONTAINER == "true" ]]; then
+            sudo apt-get update
             sudo apt-get install --no-install-recommends -y \
                 pocl-opencl-icd
+        elif [[ $(uname -m) == "aarch64" ]]; then
+            yum install -y \
+                epel-release \
+                gcc-c++ \
+                hwloc-devel \
+                sudo
+            yum install -y \
```

Comment:
Why do these `yum install` blocks need to be split up? If there isn't a specific reason, could you please combine them into a single command (with alphabetized list of libraries)?

Reply:
This is because installing `epel-release` adds a yum repository required for installing the packages in the second command. This can unfortunately not be done in a single command.

Reply:
ahhhh ok that makes sense. Thanks for the explanation.